holidays/countries/mauritius.py

```diff
@@ -0,0 +1,203 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays import HolidayBase
+from holidays.calendars import _CustomIslamicHolidays, _CustomHinduHolidays
+from holidays.calendars.gregorian import JAN, FEB, APR, MAY, JUN, JUL, AUG, SEP
+from holidays.groups import (
+    ChineseCalendarHolidays,
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+)
+
+
+class Mauritius(
+    HolidayBase,
+    ChineseCalendarHolidays,
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+):
+    """Mauritius holidays.
+
+    References:
+        * [Public Holidays Act Consolidated 1991](https://web.archive.org/web/20250618211720/https://natlex.ilo.org/dyn/natlex2/natlex2/files/download/101480/MUS101480.pdf)
+        * [The Public Holidays (Amendment) Act 2015](https://web.archive.org/web/20240805131941/http://mauritiusassembly.govmu.org/mauritiusassembly/wp-content/uploads/2023/03/act2815.pdf)
+        * Mauritius became independent in 1968, but the earliest accessible version of the Public
+          Holidays Act is the one consolidated in 1991. The most recent update to the holiday
+          schedule reflected in that version dates back to 1987. Therefore, 1988 is being used
+          as the starting year.
+        * Ougadi is another name for the Telegu Holiday, Ugadi, which is celebrated on the same
+          day as Gudi Padwa. Therefore, reusing Gudi Padwa for adding Ougadi.
+          <https://en.wikipedia.org/wiki/Ugadi>
+        * Dates for Maha Shivaratree, Ougadi and Divali match the default dates for Maha Shivratri,
+          Gudi Padwa and Diwali (India dates) in the Hindu calendar.
+            - <https://web.archive.org/web/20250322231005/https://www.timeanddate.com/holidays/mauritius/maha-shivaratree>
+            - <https://web.archive.org/web/20241012141403/https://www.timeanddate.com/holidays/mauritius/ougadi>
+            - <https://web.archive.org/web/20241112085826/https://www.timeanddate.com/holidays/mauritius/divali>
```

Comment:
```suggestion
        * [Ougadi](https://en.wikipedia.org/wiki/Ugadi) is another name for the Telugu holiday, Ugadi,
          which is celebrated on the same day as Gudi Padwa. Therefore, reusing Gudi Padwa for adding Ougadi.
```
I think the second item can be just a comment in PR.